gemrb/plugins/AREImporter/AREImporter.cpp

```diff
@@ -262,7 +262,7 @@ static TileProps MakeTileProps(const TileMap* tm, const ResRef& wedref, bool day
 		}
 		
 		if (hmit.clip.PointInside(pos)) {
-			b = hmpal->col[*hmit].r; // pick any channel, they are all the same
+			b = hmpal->operator[](*hmit).r; // pick any channel, they are all the same
```

Comment:
Can you add a GetColorAt or somesuch getter, so all these accesses will look a bit cleaner?

Reply:
Sure.